src/components/HrViewElements/FilteringModal.tsx

```diff
@@ -0,0 +1,212 @@
+                                value={"Brak preferencji"}/></>
+                    </FilteringButtonsSection>
+                </div>
+
+                {/* NUMERIC INPUT SECTION #1 */}
+                <div className="flex flex-col items-start gap-2 mt-5">
+                    <span>Oczekiwane wynagrodzenie miesięczne netto </span>
+                    <div className="flex max-sm:flex-col flex-row gap-3 items-center">
+                            <span className="flex flex-row items-center">Od&nbsp; <FilteringSalaryField
+                                registerName={"minSalary"} placeholder="Np. 1000"/></span>
+                        <span className="flex flex-row items-center">do&nbsp; <FilteringSalaryField
+                            registerName={"maxSalary"} placeholder="Np. 7000"/></span>
+                        {(methods.formState.errors.minSalary || methods.formState.errors.maxSalary) &&
+                            <span className="flex flex-col text-xs text-primary ml-6">
+                                    <span>{methods.formState.errors.minSalary?.message}</span>
+                                    <span>{methods.formState.errors.maxSalary?.message}</span>
+                                </span>}
+                    </div>
+                </div>
+
+                {/* SECTION WITH A TOGGLE SWITCH */}
+                <div className="mt-3">
+                    <label className="label cursor-pointer justify-start gap-5">
+                        <span className="">Zgoda na odbycie bezpłatnych praktyk/stażu na początek</span>
+                        <input
+                            type="checkbox" {...methods.register('canTakeApprenticeship')}
+                            style={{borderRadius: "20px"}}
+                            className="toggle toggle-primary"
+                            defaultChecked={false}/>
+                    </label>
+                </div>
+
+                {/* NUMERIC INPUT SECTION #2 */}
+                <div className="flex flex-col items-start gap-2 mt-3">
+                    <span>Ilość miesięcy doświadczenia komercyjnego kandydata w programowaniu</span>
+                    <div className="flex flex-row gap-3 items-center">
+                        <FilteringNumericalInput registerName={`monthsOfCommercialExp`} placeholder={`Np. 3`}
+                                                 min={"0"}/>
+                        <span
+                            className="text-xs text-primary ml-6">
+                                {methods.formState.errors.monthsOfCommercialExp?.message}
+                            </span>
+                    </div>
+                </div>
+
+                <div className="flex flex-row w-full items-center justify-end gap-3">
+                    <span onClick={unSetModal}
+                          className="flex items-center align-middle px-5 z-10 w-1/8 h-9 normal-case font-normal text-base rounded-none cursor-pointer hover:bg-base-100">
```

Comment:
Od razu mówię, że dla mnie to jest w miarę OK, ale jakby Marcin G. zobaczył tyle klas to by strzelał :D Takie rzeczy warto ukrywać pod konkretnymi komponentami i nie powinno być tyle klas napaćkane w różnych miejscach dużego komponentu.